src/urllib3/__init__.py

```diff
@@ -20,9 +20,9 @@
 import logging
 from logging import NullHandler
 
-__author__ = "Andrey Petrov (andrey.petrov@shazow.net)"
-__license__ = "MIT"
-__version__ = "1.25.7"
+__author__ = constants.__author__
```

Comment:
Let's revert the addition of `constants.py` and instead create a function that generates the `User-Agent` header value. Within that function you can import `__version__` without causing a cyclic import. :)

Reply:
Done in the original work.